### PR TITLE
bug 1186899 - intern browserstack improvements

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

**Comment by groovecoder** [comment]
Rebased and changed to 2 browsers: Chrome on Mac, Firefox on Windows 8. Should make the browserstack runs faster.

File: tests/ui/intern-browserstack.js [hunk 1]
```diff
@@ -3,14 +3,22 @@
...
-        { browserName: 'firefox', version: ['40'], platform: [ 'MAC' ] },
-        { browserName: 'chrome', version: ['44'], platform: [ 'MAC' ] },
+        { browserName: 'firefox', platform: ['WIN8'] },
+        { browserName: 'chrome', platform: ['MAC'] },
     ];
 
-    // Name of the tunnel class to use for WebDriver tests
-    config.tunnel = 'BrowserStackTunnel';
+    // Format for outputting test results
+    config.reporters = ['Pretty'];
```
Comment (darkwing) [inline tests/ui/intern-browserstack.js]: This is added again

**Comment by groovecoder** [comment]
Updated (and rebased again).

**Comment by darkwing** [comment]
R+, thanks Luke!

**Pull request merged**